test/Masa.Contrib.Data.IdGenerator.Snowflake.Tests/IdGeneratorTest.cs

```diff
@@ -1,8 +1,6 @@
 // Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
-using Masa.Utils.Caching.Redis;
-
```

Comment:
`The field 'IdGeneratorTest._currentWorkerKey' is assigned but its value is never used` in line 9, 10, 11, 12.